api/controllers/reservationController.js

```diff
@@ -0,0 +1,18 @@
+const reservationService = require("../services/reservationService");
+const { catchAsync } = require("../utils/error");
+
+const getOptions = catchAsync(async (req, res) => {
+  const { movies, cinema_names, hall_types, dates } = req.query;
+
+  const getOptions = await reservationService.getOptions(
+    movies,
+    cinema_names,
+    hall_types,
+    dates
+  );
+  return res.status(201).json({ getOptions });
```

Comment:
```suggestion
  return res.status(200).json({ getOptions });
```
데이터를 생성 및 수정하는 POST 메소드가 아닌 GET 메소드를 활용하고 있으며 실제 내부 로직 또한 데이터 변경이나 생성이 아닌, query 해온 값을 Return하는 것이므로 200이 적합하겠네요.